Assets/Scripts/Turret/Tracking.cs

```diff
@@ -0,0 +1,146 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Cosmobot
+{
+    public class Tracking : MonoBehaviour {
+        
+        TurretStats turretStats;
+        public GameObject target = null;
```

Comment:
This probably shouldn't be `public`. Maybe change to accessor:

```suggestion
        public GameObject target {get; private set;} = null;
```